### PR TITLE
Fix build errors

### PR DESCRIPTION
**Pull request by accept1248**
Current version causes build errors under my environment.
To fix that, I added some changes and the errors are cleared.

### CODE REVIEW

**Review by accept1248 — COMMENTED** [review]
FYR
https://developer.android.com/build/configure-app-module#set-namespace
https://stackoverflow.com/questions/76021570/execution-failed-for-task-appkaptgeneratestubsdebugkotlin

**Comment by onevcat** [comment]
We probably will keep the existing `example/pubspec.lock` as it is already in the `master` branch now. But for the changes in the other two files, I would like to invite @YkSix to evaluate, since I am not sure about it.

@YkSix Do you think we'd better contain the changes in the `build.gradle` and `AndroidManifest.xml` as suggested?

File: android/build.gradle [hunk 1]
```diff
@@ -27,7 +27,8 @@ apply plugin: 'kotlin-android'
 
 android {
     compileSdk 33
-
+    namespace "com.linecorp.flutter_line_sdk"
```
Comment (YkSix) [inline android/build.gradle:30]: maybe we can change it to 
```gradle
if (project.android.hasProperty("namespace")) {
    namespace "com.linecorp.flutter_line_sdk"
}
```

**Comment by YkSix** [comment]
@accept1248 ,
Thanks for your efforts.
Could you please provide details about your development environment, as well as the versions of Gradle and the Android Gradle Plugin (AGP) you are using?
Thanks.

**Pull request closed without merging**